deploy/kubernetes/dolphinscheduler/templates/statefulset-dolphinscheduler-master.yaml

```diff
@@ -109,11 +109,9 @@ spec:
           volumeMounts:
             - mountPath: "/opt/dolphinscheduler/logs"
               name: {{ include "dolphinscheduler.fullname" . }}-master
-            {{- include "dolphinscheduler.sharedStorage.volumeMount" . | nindent 12 }}
```

Comment:
That's fine with me. Why not remove this from `api-server` as well.

Reply:
it seems this config is needed,
I adjust the order and fix some config in `worker`